doc/code_snippets/snippets/config/instances.enabled/audit_log/myapp.lua

```diff
@@ -0,0 +1,51 @@
+-- myapp.lua --
+
+-- Create space
+function create_space()
+    box.schema.space.create('Bands')
+    box.space.bands:format({
+        { name = 'id', type = 'unsigned' },
+        { name = 'band_name', type = 'string' },
+        { name = 'year', type = 'unsigned' }
+    })
```

Comment:
I guess it doesn't work: `Bands` and `bands` are different names. There is no `bands` space.

Reply:
Fixed